app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -178,3 +178,23 @@
     }
   }
 }
+
+@include govuk-media-query($media-type: print) {
+  .gem-c-action-link {
+    &,
```

Comment:
I'm not sure this line is needed, since the line after seems to cover all the bases - there isn't any text within `.gem-c-action-link` that isn't already wrapped in a sub element (i.e. the `*`).

Reply:
I hadn't noticed that, thanks. Now fixed.